src/main/kotlin/org/rust/ide/refactoring/extractFunction/RsExtractFunctionHandler.kt

```diff
@@ -75,6 +71,27 @@ class RsExtractFunctionHandler : RefactoringActionHandler {
         }
     }
 
+    private fun findExistingInherentImpl(traitImpl: RsImplItem): RsImplItem? {
+        check(traitImpl.traitRef != null)
+        val traitCachedImpl = RsCachedImplItem.forImpl(traitImpl)
+        return (traitImpl.parent as? RsItemsOwner)
+            ?.childrenOfType<RsImplItem>()
+            ?.firstOrNull {
+                it.traitRef == null
```

Comment:
[minor] Using `org.rust.lang.core.resolve.RsCachedImplItem#isInherent` might be more consistent with the next check which also operates `RsCachedImplItem`